nunit.sln

```diff
@@ -1,6 +1,9 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 2012
+# SharpDevelop 5.1
```

Comment:
Did you make any actual changes to the solution or is this just SharpDevelop re-arranging things? Any added projects?